pkg/dataprotection/backup/utils.go

```diff
@@ -302,3 +311,16 @@ func StopStatefulSetsWhenFailed(ctx context.Context, cli client.Client, backup *
 	sts.Spec.Replicas = pointer.Int32(0)
 	return cli.Update(ctx, sts)
 }
+
+func BuildParametersManifest(parameters []dpv1alpha1.ParameterPair) string {
+	var res string
+	if len(parameters) == 0 {
+		return res
+	}
+	for _, v := range parameters {
+		res += fmt.Sprintf("  - name: %s\n", v.Name)
+		res += fmt.Sprintf("    value: %s\n", v.Value)
```

Comment:
The value is not properly escaped and it is possible to construct an invalid YAML content. Since YAML is a superset of JSON, I suggest using Go's JSON encoder to encode the whole parameters. For example, the render result should be like:

```
  parameters: [{"name":"name1","value":"value1"},{"name":"name2","value":"value2"}]
```

Reply:
done